### DISCUSSION TITLE
Any way to make the ScrollArea's content div to fill the page?

### DISCUSSION THREAD

**Discussion by ShookLyngs** [mantinedev/mantine#2418] [Q&A]
Right now, our content div in ScrollAreaViewport has no options for css/style customization.

so if I want the content div to fill the ScrollArea's height by default, right now it seems we can't?
Or if I'm wrong, please correct me.

Example layout (with TailwindCSS):
```tsx
<div className="absolute inset-0 flex-flex-col">
  <Header/>
  <ScrollArea>
    {// want this part to fill all remaining space, so the below content can show at the center of the page}
    <div className="flex-auto flex justify-center items-center">404</div>
  </ScrollArea>
</div>
```

**Comment by rtivital** [comment]
https://codesandbox.io/s/solitary-water-b80lvp?file=/src/App.tsx

**Reply by ShookLyngs** [reply]
> https://codesandbox.io/s/solitary-water-b80lvp?file=/src/App.tsx

In my case the `const content = Array(100)` part should be `const content = Array(1)`, and the "content" text should be displayed at the center (vertically and horizontally) of the rest of the page. 

Any solution to this problem?

**Comment by ShookLyngs** [comment]
Okay, did it with imperfect css like this:
```postcss
.ScrollAreaViewport--full {
  @apply flex flex-col flex-auto;

  // the content div
  & > div {
    @apply flex-col flex-auto;
    display: flex !important;
  }
}
```
